website/blog/2021-02-19-tungsten.md

```diff
@@ -0,0 +1,498 @@
+---
+author: Tomasz Godzik
+title: Metals v0.10.0 - Tungsten
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.10.0, which is the first
+release in the "Tungsten" line. In `0.10.x` we will focus on polishing existing
+features, fixing bugs and improving the Scala 3 support at least to the level of
+Scala 2. This will pave the way towards the `1.0.0` release later this year.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">156</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">74</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">30</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">10</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/34?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Type decorations for definitions.
+- Insert inferred type refactoring.
+- Configuration option for the default Scala version.
+- Code lens to easily discover the copy worksheet command.
+- Improved short names when importing.
+- Debug run/test when using sbt via BSP.
+- Support for Scala 3.0.0-RC1
```

Comment:
Yeah, I didn't want to duplicate it, but I think it might be sensible to make it as explicit as possible. We've had numerous questions about it previously. Will add!